ref/general/securityUtility.adoc

```diff
@@ -0,0 +1,206 @@
+//
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: The securityUtility command supports plain text encryption and TLS certificate creation for Open Liberty.
+:seo-title: securityUtility command
+:seo-description: The securityUtility command supports plain text encryption and TLS certificate creation for Open Liberty.
+:page-layout: general-reference
+:page-type: general
+= securityUtility command
+
+The `securityUtility` command supports plain text encryption and TLS certificate creation for Open Liberty.
+
+You can call the tool directly from the command line or allow the server to call it to create a default TLS certificate and keystore on server startup.
```

Comment:
I think to avoid any possible confusion, you could maybe change "allow the server to call it" to "allow the server to call the tool" to maintain the clarity of the subject